Cue/app/tabs/search/SearchListView.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+
+'use strict';
+
+import React from 'react'
+import { Navigator, View, Text, ListView, Image, Platform, Dimensions } from 'react-native'
+
+import { connect } from 'react-redux'
+
+import type { DeckMetadata, Card } from '../../api/types'
+
+import EmptyView from '../../common/EmptyView'
+import CueIcons from '../../common/CueIcons'
+
+import SearchListViewItem from './SearchListViewItem'
+import ListViewHairlineSeparator from '../../common/ListViewHairlineSeparator'
+
+const styles = {
+}
+
+type Props = {
+  navigator: Navigator,
+  decks: Array<DeckMetadata>,
+}
+
+class SearchListView extends React.Component {
+  props: Props
+
+  ds: ListView.DataSource
+
+  state: {
+    dataSource: ListView.DataSource,
+    deviceOrientation: string
+  }
+
+  _onLayout = () => {
+    const windowDimensions = Dimensions.get('window')
+    this.setState({
+      ...this.state,
+      deviceOrientation: windowDimensions.width > windowDimensions.height ? 'LANDSCAPE' : 'PORTRAIT'
+    })
+  }
+
+  constructor(props: Props) {
+    super(props)
+
+    this.ds = new ListView.DataSource({
+      rowHasChanged: (r1, r2) => r1 !== r2
+    })
+
+    this.state = {
+      deviceOrientation: 'UNKNOWN',
+      dataSource: this.ds.cloneWithRows(props.decks)
+    }
+  }
+
+  componentWillReceiveProps(props) {
+    this.setState({
+      ...this.state,
+      dataSource: this.ds.cloneWithRows(props.decks)
+    })
+  }
+
+  render() {
+    if (this.state.dataSource.getRowCount() == 0) {

```

Comment:
Can you create a separate issue to display a blank view instead of the no matches view if a search hasn't been performed yet?